goodtables.yml

```diff
@@ -0,0 +1,9 @@
+datapackages:
+  - DataPackages/Goal/1/datapackage.json
+  # - DataPackages/Goal/2/datapackage.json
+  # - etc
+
+settings:
+  row_limit: 100000
+  checks:
+    duplicate-row: false
```

Comment:
https://goodtables.io/github/roll/SDG/jobs/8

As we can see `Goal/1/dimensionvalues.csv` has 306 duplicating rows (click on `Duplicate Row`) for more information. For some datasets it's ok - so I disabled checking duplicated rows. If it's not OK we should fix the data.